build-scripts/webpack.cjs

```diff
@@ -84,6 +84,15 @@ const createWebpackConfig = ({
       ],
       moduleIds: isProdBuild && !isStatsBuild ? "deterministic" : "named",
       chunkIds: isProdBuild && !isStatsBuild ? "deterministic" : "named",
+      splitChunks: {
+        // Disable splitting for web workers with ESM output
+        // Imports of external chunks are broken
+        chunks: latestBuild
+          ? (chunk) =>
+              !chunk.canBeInitial() &&
+              !/(sort_filter|markdown)_worker/.test(chunk.name)
```

Comment:
Can we make this more generic, so that if a worker is added it will work for that too? Like name all our workers `name.worker.ts`?

Reply:
Done.  New workers just need to be given a chunk name ending in "-worker".